src/components/Button/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {Button} from 'react-native';
+
+CustomButton = ({value, style, color, onPress}) => {
```

Comment:
you are not using any data-type in front of CustomButton like var, const. Isn't it showing any error/warning? Explore it too.

Reply:
There was no error or warning. I will add the data type and check again.